src/db-up/azext_db_up/commands.py

```diff
@@ -32,20 +37,27 @@ def load_command_table(self, _):  # pylint: disable=too-many-locals, too-many-st
         from ._exception_handler import password_handler
         g.custom_command('up', 'mysql_up', validator=db_up_namespace_processor('mysql'),
                          table_transformer=table_transform_connection_string,
-                         exception_handler=password_handler)
+                         exception_handler=password_handler,
+                         deprecate_info=g.deprecate(message_func=_deprecation_message))
```

Comment:
Is it possible to set `deprecate_info` at command group `mysql` level?

Reply:
It's not appropriate because `az mysql` is also defined in `Mysql` module in cli main repo and it's not going to be deprecated. That's why I add the deprecation for each command